### ISSUE TITLE
Show map with location above the Get Directions link in the booking modal

### ISSUE THREAD

**Issue by olafwrieden** [CovidEngine/vaxxnz#141]
As a user, I'd like to see the location of the vaccination location on a map in the modal (above `Get Directions`) so that I can easily find the location.

**Some ideas**

- `iFrame` map render with Mapbox
- Pained path from current location to destination
- Open to ideas

**Comment by nzjs** [comment]
Mapbox GL would be a good option as they are [actively supporting COVID-19 related applications](https://blog.mapbox.com/support-for-covid-19-apps-497de40fee9c), both financially and with technical support. 

You could also use their [Directions API](https://docs.mapbox.com/help/getting-started/directions/) to display a route from the current location to selected destination.

Another option for a mapping library would be [MapLibre GL](https://github.com/maplibre/maplibre-gl-js), but this becomes a little more involved to style and setup your own vector basemaps.

**Comment by noway** [comment]
@nzjs that's awesome! We're already using Google Maps iframe embed here though (https://vaxx.nz/locations/the-auckland-city-doctors-2076593017), that one is actually free. If we just do the map without directions, that would be an easier solution!

Would love for you take on that task and tackle it :-)

**Comment by nzjs** [comment]
@noway Sure I can take a look at this, I hadn't noticed that embed. Is the Google API costing anything right now?

So to clarify, you would like the current G Maps iframe replaced with one of the other mapping APIs as a component.
Referring to this: https://github.com/CovidEngine/vaxxnz/blob/main/src/today-locations/healthpoint/HealthpointModal.tsx#L200-L209 

Or are we talking about adding a new small map component near the Get Directions link, in this modal?
![snip](https://user-images.githubusercontent.com/5310559/135001087-87739386-4c11-43f7-9bb4-cba2b1e66eab.png)

**Comment by noway** [comment]
Yeah, adding a small map there. It does mean we'll have a lot of small maps on that page @waltzaround is that a good idea?
